tests/population_tests.c

```diff
@@ -347,8 +347,6 @@ void test_population_next_generation()
     /* TODO plot the csv file */
 
 
-    sprintf(commandstr,"cp %s /home/bashrc/develop/scalam/test.txt", log_file);
-    run_shell_command(commandstr);
 
     /* remove the log file */
     sprintf(commandstr,"rm -f %s", log_file);
```

Comment:
Although I pointed out the absolute path, surely we need to make sure we can actually access the file that is produced? The next few lines remove this file